### PR TITLE
New refactoring: Replace one method call by another one

### LINKED ISSUE #4320: Missing refactoring - Replace one method call by another one
May be we have it.

But how to replace one method call by another one

for example we have two methods
atX: x atY: y
and
atRow: rowNumber atColumn: columnNumber
and we want all methods to use atRow:atColumn:

https://github.com/Lin777/Enjoliveur/issues/8

### PR DESCRIPTION
**Pull request by Lin777**
Fixes #4320 

Link video: https://www.youtube.com/watch?v=CJjZYugsqZg&feature=youtu.be

### CODE REVIEW

**Comment by Ducasse** [comment]
super cool. 
Now no need of parentheses around

senders := (self model allReferencesTo: oldSelector)

and 

(self new)

File: src/Refactoring-Core/RBReplaceCalledMethodByAnotherRefactoring.class.st [hunk 1]
```diff
@@ -0,0 +1,72 @@
+"
+I'm a refactoring operation for replace one method call by another one.
+
+The another method name should have the same number of arguments.
+
+All senders of this method are changed by the other
+"
+Class {
+	#name : #RBReplaceCalledMethodByAnotherRefactoring,
```
Comment (dionisiydk) [inline src/Refactoring-Core/RBReplaceCalledMethodByAnotherRefactoring.class.st]: I think class names can be simplified. Simple RBReplaceMethodRefactoring would be good enough IMHO

File: src/SystemCommands-MessageCommands/SycReplaceMessageCallWithOtherCommand.class.st [hunk 2]
```diff
@@ -0,0 +1,53 @@
+Class {
+	#name : #SycReplaceMessageCallWithOtherCommand,
```
Comment (dionisiydk) [inline src/SystemCommands-MessageCommands/SycReplaceMessageCallWithOtherCommand.class.st]: And here it would be SycReplaceMessageCommand

**Review by dionisiydk — CHANGES_REQUESTED** [review]

**Comment by Lin777** [comment]
Great, I already corrected the suggestions :D

File: src/SystemCommands-MessageCommands/SycReplaceMessageCallWithOtherCommand.class.st (hunk 2 above)
Comment (Lin777) [reply src/SystemCommands-MessageCommands/SycReplaceMessageCallWithOtherCommand.class.st]: Thanks, I already changed the names

**Comment by Ducasse** [comment]
Super cool!.
Tx

**Pull request merged**